clients/filesystem-hadoop3/src/main/java/org/apache/gravitino/filesystem/hadoop/GravitinoVirtualFileSystem.java

```diff
@@ -125,13 +136,33 @@ public void initialize(URI name, Configuration configuration) throws IOException
 
     initializeClient(configuration);
 
+    initializePluginFileSystem(configuration);
+
     this.workingDirectory = new Path(name);
     this.uri = URI.create(name.getScheme() + "://" + name.getAuthority());
 
     setConf(configuration);
     super.initialize(uri, getConf());
   }
 
+  private void initializePluginFileSystem(Configuration configuration) {
+    String fileSystemProviders = configuration.get("fs.gvfs.filesystem.providers");
```

Comment:
I think you need define this config in the `GravitinoVirtualFileSystemConfiguration`, and how about use `fs.gravitino.filesystem.providers`.

Reply:
> I think you need define this config in the GravitinoVirtualFileSystemConfiguration

OK.

> and how about use fs.gravitino.filesystem.providers

I will use another PR to add some documents about it after S3 and GCS are finished, as GVFS doesn't depend on this configuration in the current code base to support HDFS and localfile.